backend/controllers/attendance/activityList.ts

```diff
@@ -17,18 +16,25 @@ const sortActivitiesByEventThenByLastName = (data) => {
   })
 }
 
-const extractAttendanceInfo = (attendanceInformation, event, absentReasons = []) => {
+const extractAttendanceInfo = (attendanceInformation, event) => {
   if (attendanceInformation && attendanceInformation.attendances && attendanceInformation.attendances.length > 0) {
     const offenderAttendanceInfo = attendanceInformation.attendances.find(
       (attendance) => attendance.bookingId === event.bookingId && attendance.eventId === event.eventId
     )
     if (!offenderAttendanceInfo) return null
 
-    const { id, absentReason, absentSubReason, attended, paid, comments, locked } = offenderAttendanceInfo || {}
-    const mapToAbsentReason = absentReasonMapper(absentReasons)
+    const { id, absentReason, absentReasonDescription, absentSubReason, attended, paid, comments, locked } =
```

Comment:
grab description from whereabouts api and use that instead of converting the reason